nominees/covid-19-scenarios.json

```diff
@@ -10,9 +13,14 @@
   ],
   "SDGs": [
     {
-      "SDGNumber": 3
+      "SDGNumber": 3,
+      "evidenceText": "Pandemic planning has been proven over the past two years to be a critical component of public health. COVID-19 scenarios directly serves this need by allowing experts to forecast infections and hospital demand and subsequently disseminate their results and assumptions openly. ",
+      "evidenceURL": "https://github.com/neherlab/covid19_scenarios/blob/master/LICENSE"
```

Comment:
This should be the URL for the license, not evidence

Reply:
Resolved in commit db7342a